python_modules/dagster/dagster_tests/daemon_tests/test_backfill.py

```diff
@@ -613,6 +639,101 @@ def test_simple_backfill(
     assert three.tags[PARTITION_NAME_TAG] == "three"
 
 
+@pytest.mark.parametrize("parallel", [True, False])
+def test_two_backfills_at_the_same_time(
+    tmp_path: Path,
+    parallel: bool,
+):
+    with instance_for_test(
+        overrides={
+            "event_log_storage": {
+                "module": "dagster._core.storage.event_log",
+                "class": "ConsolidatedSqliteEventLogStorage",
+                "config": {"base_dir": str(tmp_path)},
+            },
+            "run_retries": {"enabled": True},
+        }
+    ) as instance:
+        with create_test_daemon_workspace_context(
+            workspace_load_target=ModuleTarget(
+                module_name="dagster_tests.daemon_tests.test_backfill",
+                attribute="the_repo",
+                working_directory=os.path.join(os.path.dirname(__file__), "..", ".."),
+                location_name="test_location",
+            ),
+            instance=instance,
```

Comment:
maybe worth a comment here about why we can't use the same fixtures that the rest of the tests are using - we could also consider still pulling these out into fixtures , just different fixtures (thread_safe_instance / thread_safe_workspace_context?)

Reply:
If you're fine with it, maybe I'll just add the comment and not pull it into separate fixtures for now, since the fixtures wouldn't be used for anything else at this time?